src/reg.c

```diff
@@ -229,10 +229,18 @@ int reg_register(struct reg *reg, const char *reg_uri, const char *params,
 	if (acc && acc->fbregint)
 		err = sipreg_set_fbregint(reg->sipreg, acc->fbregint);
 
+	if (acc && acc->tcpsrcport)
+		err = sipreg_set_srcport(reg->sipreg, acc->tcpsrcport);
```

Comment:
can this function fail ?

Reply:
Not really. I will change the function to return void.